extensions/positron-python/python_files/posit/positron/tests/test_variables.py

```diff
@@ -97,41 +97,45 @@ def test_comm_open(kernel: PositronIPyKernel) -> None:
             ],
         ),
         # Nested mutable types
-        ("", ["x = [{}]", "x[0]['a'] = 0"]),
-        ("", ["x = {'a': []}", "x['a'].append(0)"]),
+        ("", [" = [{}]", "[0]['a'] = 0"]),
+        ("", [" = {'a': []}", "['a'].append(0)"]),
     ],
 )
+@pytest.mark.parametrize("varname", ["x", "_"])
```

Comment:
Woah, I did not know that you can chain `parametrize`, very useful!

Reply:
yeah at my last job I was guilty of writing a test with like 6 different parametrizations resulting in like 3500 cases 😂 coverage was really important!!